assets/css/style.scss

```diff
@@ -1,7 +1,10 @@
-@use './modules/docs';
-@use './modules/base';
-@use './modules/genericons';
-@use './modules/menu';
-@use './modules/middleware';
-@use './modules/navbar';
-@use './modules/syntax';
+---
+---
+
+@import 'docs';
+@import 'base';
+@import 'genericons';
+@import 'menu';
+@import 'middleware';
+@import 'navbar';
+@import 'syntax';
```

Comment:
@ikbensiep  Although `@use` is better, github pages uses an older version of sass that doesn't support it, so I went back to `@import`